docs/run-random-beacon.adoc

```diff
@@ -43,6 +43,35 @@ Keep Random Beacon client.
 |2 vCPU / 4 GiB RAM / 1 GiB Persistent Storage
 |===
 
+== Gas Costs
+
+Random Beacon smart contracts reimburse the operator for successfully submitting relay entry and DKG result but they do not reimburse for submitting a group selection ticket. Reimbursements are sent to the beneficiary account and can be claimed along with rewards once the group expires. It is expected that the operators have enough ETH on the accounts used by clients to submit the required transactions and that the operator account balance is monitored and refilled as needed. Bear in mind that the higher stake is, the operator is selected more frequently and is expected to submit more transactions as a result.
```

Comment:
Let's line wrap here and in the next paragraph.

Reply:
Maybe we can also clarify (could be its own paragraph after the table, not sure) that reimbursements happen after successful group formation?

Reply:
> Maybe we can also clarify (could be its own paragraph after the table, not sure) that reimbursements happen after successful group formation?

We do have:

> Random Beacon smart contracts reimburse the operator for successfully submitting relay entry and DKG result

🤔 Isn't it clear from this sentence?